apps/server/src/modules/authorization/authorization.service.ts

```diff
@@ -74,7 +76,7 @@ export class AuthorizationService extends BasePermissionManager {
 		context: IPermissionContext
 	) {
 		if (!(await this.hasPermissionByReferences(userId, entityName, entityId, context))) {
-			throw new ForbiddenException();
+			throw new ForbiddenError(userId, entityId, context);
```

Comment:
why throw an error here and a loggable above?

Reply:
As commented in the respective classes, those are alternative implementations.